test/recipes/15-test_rsapss.t

```diff
@@ -64,3 +64,11 @@ ok(run(app(['openssl', 'dgst', '-prverify', srctop_file('test', 'testrsa.pem'),
             '-signature', 'testrsapss-unrestricted.sig',
             srctop_file('test', 'testrsa.pem')])),
    "openssl dgst -prverify [plain RSA key, PSS padding mode, no PSS restrictions]");
+
+
+# Test for issue #17167
```

Comment:
Please instead of using an issue number rather describe what the test does: `Test that RSA-PSS keys are supported by genpkey and rsa commands.`

Nit: Also no need to have two empty lines to separate, one would be sufficient.